packages/css/src/components/form/fieldset.scss

```diff
@@ -0,0 +1,48 @@
+@use "sass:math";
```

Comment:
Waarom heb je deze file gemaakt? Code organisatie of omdat je in de knoop zat met `@use` / `@import` volgordes?

Reply:
OK!

Reply:
Maar je maakt geen gebruik van `math` functions